.augment/commands/bmad/workflows/bmm-solutioning-gate-check.md

```
@@ -0,0 +1,50 @@
+# Module path and component files
+installed_path: "{project-root}/bmad/bmm/workflows/3-solutioning/solutioning-gate-check"
+template: "{installed_path}/template.md"
+instructions: "{installed_path}/instructions.md"
+validation: "{installed_path}/checklist.md"
+
+# Output configuration
+default_output_file: "{output_folder}/implementation-readiness-report-{{date}}.md"
+
+# Expected input documents (varies by project level)
+recommended_inputs:
+  - prd: "{output_folder}/prd*.md"
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)